src/comp/comp.py

```diff
@@ -24,48 +24,50 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [i.name for i in humans if str(i.name).startswith('D')]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [i.name for i in humans if str(i.name).endswith('e')]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+starters = ['C', 'D', 'E', 'F', 'G']
+c = [i.name for i in humans if i.name[0] in starters]
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = [i.age + 10 for i in humans]
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [(f"{i.name}-{i.age}") for i in humans]
 print(e)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
-f = []
+#f = [f'("{i.name}", {i.age})' for i in humans if (i.age > 26) & (i.age < 33)]
+f = [tuple(f'{i.name}, {i.age}') for i in humans if (i.age > 26) & (i.age < 33)]
 print(f)
 
 # Write a list comprehension that creates a list of new Humans like the old
 # list, except with all the names uppercase and the ages with 5 added to them.
 # The "humans" list should be unmodified.
 print("All names uppercase:")
-g = []
+g = [(i.age + 5, str(i.name).upper()) for i in humans]
```

Comment:
Create new Human class objects, not tuples:

`g = [Human(i.name.upper(), i.age + 5) for i in humans]`